python/lsst/imgserv/skymapStitch.py

```diff
@@ -86,29 +85,82 @@ def getSkyMap(ctrCoord, width, height, filt, source, mapType, patchType):
             patchImg = patchExposure.getMaskedImage()
             srcViewImg <<= patchImg
 
-    # Copy the pixels from the patches into the destination exposures.
+    # Copy the pixels from the source exposures to the destination exposures.
     destExposureList = []
-    for srcExpo in srcExposureList:
+    for j, srcExpo in enumerate(srcExposureList):
         sImg = srcExpo.getMaskedImage()
         srcWcs = srcExpo.getWcs()
-        dExpo = afwImage.ExposureF(destBBox, srcWcs)
+        if j == 0:
+            dBBox = destBBox # destBBox only correct for first image
+        else:
+            # Determine the correct BBox (in pixels) for the current srcWcs
+            llCorner = afwGeom.Point2I(srcWcs.skyToPixel(destCornerCoords[0]))
+            urCorner = afwGeom.Point2I(srcWcs.skyToPixel(destCornerCoords[2]))
+            # Handle negative values for dBBox llCorner.
+            if llCorner.getX() < 0:
+                llCorner.setX(0)
+            if llCorner.getY() < 0:
+                llCorner.setY(0)
```

Comment:
urCorner being larger than the image should be handled below with the calculations for endX